### PR TITLE
Add count to repair queue

### PR DESCRIPTION
**Pull request by aleitner**
What: 
Add method for counting number of segments in repair queue

Why:
We need to test if the repair queue removes segments when expected, but there is currently no way to look into the repair queue. Count can also provide metrics on how the queue is working.

Please describe the tests:
 - Test 1: Add 100 items to the queue and count to verify 100 items were added

 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/satellitedb/repairqueue.go [hunk 1]
```diff
@@ -117,3 +117,22 @@ func (r *repairQueue) SelectN(ctx context.Context, limit int) (segs []pb.Injured
 	}
 	return segs, Error.Wrap(rows.Err())
 }
+
```
Comment (mobyvb) [inline satellite/satellitedb/repairqueue.go]: add a comment here. Make sure to clarify that this counts every segment, regardless of how recently it was attempted

Comment (aleitner) [reply satellite/satellitedb/repairqueue.go]: done

File: satellite/satellitedb/repairqueue.go [hunk 2]
```diff
@@ -117,3 +117,23 @@ func (r *repairQueue) SelectN(ctx context.Context, limit int) (segs []pb.Injured
...
+func (r *repairQueue) Count(ctx context.Context) (count int, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	// Count every segment regardless of how recently repair was last attempted
+	rows, err := r.db.QueryContext(ctx, r.db.Rebind(`SELECT COUNT(*) as count FROM injuredsegments`))
```
Comment (egonelbre) [inline satellite/satellitedb/repairqueue.go]: `QueryRowContext`

Comment (aleitner) [reply satellite/satellitedb/repairqueue.go]: done

File: satellite/satellitedb/repairqueue.go [hunk 3]
```diff
@@ -117,3 +117,15 @@ func (r *repairQueue) SelectN(ctx context.Context, limit int) (segs []pb.Injured
...
+func (r *repairQueue) Count(ctx context.Context) (count int, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	// Count every segment regardless of how recently repair was last attempted
+	err = r.db.QueryRowContext(ctx, r.db.Rebind(`SELECT COUNT(*) as count FROM injuredsegments`)).Scan(&count)
+	if err != nil {
+		return count, Error.Wrap(err)
+	}
+
+	return count, nil
```
Comment (egonelbre) [inline satellite/satellitedb/repairqueue.go]: return count, Error.Wrap(err)

**Pull request merged**